eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -351,9 +351,6 @@ the main ServiceBusClientBuilder. -->
 
   <!-- FIXME temp -->
   <suppress checks="ServiceClient" files="com.azure.messaging.webpubsub.*"/>
-  <!-- Suppress ReverseProxyPolicy being package private. It may get moved into azure-core. -->
-  <suppress checks="com.azure.tools.checkstyle.checks.HttpPipelinePolicy"
-            files="com.azure.messaging.webpubsub.ReverseProxyPolicy.java"/>
```

Comment:
Any idea why you revert my change?

Reply:
I just ran the patch release script and that is it, there was not any manual work except changing the date release in CHANGELOG.md

Reply:
I don't think patch release script will touch this file. And there appears to be other changes to this file in your PR. Do you want me to loop in Wes on how this happens?

Reply:
yes please, if it requires I could create a PR which reverts these changes

Reply:
@weshaggard 

Hello Wes, would the "patch release script" change this checkstyle suppression file under `eng/code-quality-reports`? (I thought no, but I am no expert. And the some change was merged weeks ago, well before the GA code freeze date.)

If this should not happen, would it help if the author revert the change on this file?

Reply:
I would not expect it to touch the other files either perhaps there was some back merge or sync point. @pallavit could you please have a look to understand what might have happened here.

In the mean time we should probably undo any changes that aren't related to the patch update.

Reply:
@AikoBB - Looks like the patch script was used to prepare release from master? Were their changes done to the SDK sources in master that you preferred using patch-script vs prepare-release directly?

The patch script resets sources of the SDK and some code quality check files to bring SDK changes at par with the release tag provided. The idea is to ensure that pipeline checks can be done correctly. Since code quality files are common, the reset can impact other SDKs. The idea is NOT to check those changes in at the time of CR (but I can see why it may cause confusion or may slip through). Let me see how I can make this better.

Reply:
@pallavit Yes, the patch script(Generate-Patch.ps) was used to prepare patch release from master and ran it without any changes to the master branch